src/Form/CocktailFormType.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace App\Form;
+
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\Form\Extension\Core\Type\TextType;
+use App\Entity\CocktailName;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+class CocktailFormType extends AbstractType
```

Comment:
You could leave CocktailType as the classname as you already are in the Form folder so we get the context.

Reply:
fixed